src/compiler/diagnosticMessages.json

```diff
@@ -6342,5 +6342,9 @@
     "Invalid value for 'jsxFragmentFactory'. '{0}' is not a valid identifier or qualified-name.": {
         "category": "Error",
         "code": 18035
+    },
+    "Computed property names can only be named with well-known symbols without the 'typeof' operator. Try replacing 'typeof {0}' with '{0}'.": {
+        "category": "Error",
+        "code": 18036
     }

```

Comment:
I wasn't super sure where to put this. Do we always put new errors at the end of this file with an incremented status code or should I move this closer to the related errors?

Reply:
We do try to put them in the closest section to a relevant error typically